src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/ProgrammableAndroid.java

```diff
@@ -534,12 +534,12 @@ protected void editInstruction(Player p, Block b, String[] script, int index) {
     }
 
     @Nonnull
-    protected String getScript(@Nonnull Location l) {
+    public String getScript(@Nonnull Location l) {
```

Comment:
Don't open things up that don't have sufficient parameter validation, the `@Nonnull` is not enforced here because it was an internal method, so you'd need to add a Validate call here.